src/main/resources/org/jenkinsci/plugins/pipeline/utility/steps/conf/WriteYamlStep/help.html

```diff
@@ -50,6 +50,10 @@
             <li>UTF-16</li>
         </ul>
     </li>
+    <li>
+        <code>overwrite</code> <i>(optional)</i>:
+        Allow existing files to be overwritten. Defaults to <code>true</code>.
```

Comment:
From what I can see of the code the last sentence is a lie.

```suggestion
        Allow existing files to be overwritten. Defaults to <code>false</code>.
```

Reply:
I was initially planning to also add this for the other write steps, where the default would have been false. Apparently I edited the wrong help file. 🤦